.devcontainer/All/Dockerfile.All.SRC

```
@@ -182,5 +184,21 @@ RUN apt-get autoremove -y \
     && apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Create the non-root user and open up read/execute access to the toolchains
+# and repositories that were downloaded/cloned as root above.
+RUN groupadd --gid $USER_GID $USERNAME \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)